docs/recipes.rst

```diff
@@ -142,7 +142,7 @@ Add a comment on a pull request when it is created
           - do: comment
             payload:
                 body: >
```

Comment:
Shouldn't this be dedented instead? This seems to have an indentation of 4, when the rest of the file is 2. (Also probably my mistake, sorry)

Reply:
Good catch. I'll update.